exo-run/src/cli.ls

```diff
@@ -16,7 +16,10 @@ module.exports = ->
 
   app-config = yaml.safe-load fs.read-file-sync('application.yml', 'utf8')
   console.log "Running #{green app-config.name} #{cyan app-config.version}\n"
-  logger = new Logger flatten [Object.keys(app-config.services[type]) for type of app-config.services]
+  services = flatten [Object.keys(app-config.services[type]) for type of app-config.services]
+  silenced-services = [service if app-config.services[type][service].silent for type of app-config.services for service of app-config.services[type]]
```

Comment:
It might be worth doing a single nested for loop to build `services` and `silenced-services`, makes it a bit easier to understand what's going on.